### PR TITLE
HHVM 4.27 announcement

### PR DESCRIPTION
**Pull request by fredemmott**

### CODE REVIEW

File: _posts/2019-10-14-hhvm-4.27.0.markdown [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+4.22&ndash;4.26 remain supported, as do the LTS releases 3.30 and 4.8.
+
+# Highlights
+
+- Clearer typechecker error message when a function does not explicitly
+  return a value in all branches
+- Added 'jump-to-definition' for overridden methods
+- go-to-implementation ("textDocument/implementation") is now implemented in
+  LSP-based editors (e.g. Visual Studio Code) for common situations.
+- added `set_pre_timeout_handler(int $seconds,
```
Comment (jjergus) [inline _posts/2019-10-14-hhvm-4.27.0.markdown]: missing end of the header

File: _posts/2019-10-14-hhvm-4.27.0.markdown [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+- `break` and `continue` with a level (e.g. `break 3;`) is now a parse error;
+  previously, a typechecker error was raised
+- usage of abstract type constants now raises a type error in some places where
+  they do not have meaning (e.g.
+  `abstract class C { abstract const type T; } function foo(C::T $x)`)
+
+# Future Changes
+
+- We aim to drop support for MacOS High Sierra (10.13) and introduce support for
+  MacOS Catalina (10.15) as soon Catalina is available from our build
```
Comment (jjergus) [inline _posts/2019-10-14-hhvm-4.27.0.markdown]: as soon *as*

File: _posts/2019-10-14-hhvm-4.27.0.markdown [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+---
+
+HHVM 4.27 is released! This release marks the end of support for 4.21;
+4.22&ndash;4.26 remain supported, as do the LTS releases 3.30 and 4.8.
+
+# Highlights
+
+- Clearer typechecker error message when a function does not explicitly
+  return a value in all branches
+- Added 'jump-to-definition' for overridden methods
```
Comment (jjergus) [inline _posts/2019-10-14-hhvm-4.27.0.markdown]: maybe mention this is LSP-related (or just merge with the one below?)

File: _posts/2019-10-14-hhvm-4.27.0.markdown (hunk 1 above)
Comment (fredemmott) [reply _posts/2019-10-14-hhvm-4.27.0.markdown]: whoops, leaving to next week for better HHI

**Comment by fredemmott** [comment]
Thanks; suggested changes applied and merged as 1d43d4d

**Pull request closed without merging**